composer.json

```diff
@@ -56,7 +56,7 @@
             ]
         },
         "branch-alias": {
-            "dev-master": "3.0-dev"
+            "dev-laravel-58": "3.1-dev"
```

Comment:
I am not sure if it should not be version 4, because of the media package required as that package is backward not compatible.